apiserver/facades/client/application/application_unit_test.go

```diff
@@ -1198,3 +1198,15 @@ func (s *ApplicationSuite) TestCAASExposeWithHostname(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	app.CheckCallNames(c, "ApplicationConfig", "SetExposed")
 }
+
+func (s *ApplicationSuite) TestApplicationUnitResolved(c *gc.C) {
+	p := params.Resolved{
+		UnitName: "postgresql/0",
+		Retry:    false,
+		All:      false,
+	}
+
+	s.backend.SetErrors(nil)
+	err := s.api.Resolved(p)
+	c.Assert(err, gc.IsNil)
```

Comment:
We need a little more than this. We need to check that the facade has actually done what it's supposed to have done. The point of using the stub inside the mock is so we record the api calls made.